woocommerce/woocommerce-paypal-payments
# Show CC gateway in admin payments list

### Description

Adds CC gateway in the payments list in WC settings, to allow changing order, enabling/disabling it separately.

https://user-images.githubusercontent.com/5680466/131116506-040028b1-fd87-462a-9900-e8f087dfe232.mp4

However, currently it is not possible to disable the PayPal gateway and continue using the CC gateway. I am not sure what was the reason for that and whether we can make them separate now.

### Steps to test:
1. Open payments in WC settings.
2. Try changing order, enabling/disabling, editing settings.
3. Check that everything changes as expected and reflected during the user checkout.

### Changelog entry

- Added the PayPal Credit Card gateway in the payments list in WC settings to allow changing order.
